### DISCUSSION TITLE
Add support for attaching files to messages

### DISCUSSION THREAD

**Discussion by hseg** [d99kris/nchat#195] [Ideas]
From docs, it doesn't appear there's a keybinding for this, though there does appear to be a setting `attachment_send_type` suggesting this feature exists?

**Comment by d99kris** [comment]
You're right, this is not currently supported. Taking note of the feature request though.

**Comment by d99kris** [comment]
In 650d4cb support for sending captions along with files has been added. First one enters the text in the regular message entry field, then press `Ctrl-t` (for transfer file), and select a file. The message sent will include the file and the caption text.

The functionality can be disabled with config `transfer_send_caption=0`.

**Comment by hseg** [comment]
Just tested this, thanks!
